src/client/ui/selector-inspector-panel/selector-input-container.js

```diff
@@ -0,0 +1,162 @@
+/* eslint-disable no-restricted-properties */
+import hammerhead from './../deps/hammerhead';
+import testCafeCore from './../deps/testcafe-core';
+
+import { createElementFromDescriptor } from './utils/create-element-from-descriptor';
+import { getElementsBySelector } from './utils/get-elements-by-selector';
+
+import * as descriptors from './descriptors';
+import { elementPicker, ELEMENT_PICKED } from './element-picker';
+import { highlighter } from './highlighter';
+import {
+    selectorsList,
+    SELECTOR_SELECTED,
+    LIST_CHANGED,
+} from './selectors-list';
+
+const nativeMethods = hammerhead.nativeMethods;
+const shadowUI      = hammerhead.shadowUI;
+
+const eventUtils = testCafeCore.eventUtils;
+
+const ENABLED_CLASS = 'enabled';
+
+const MATCH_INDICATOR_CLASSES = {
+    notFound: 'not-found',
+    invalid:  'invalid',
+    ok:       'ok',
+};
+
+export class SelectorInputContainer {
+    element;
+    input;
+    indicator;
+    expandButton;
+
+    constructor () {
+        this._createElements();
+        this._addEventListeners();
+    }
+
+    get value () {
+        return nativeMethods.inputValueGetter.call(this.input);
+    }
+
+    set value (value) {
+        nativeMethods.inputValueSetter.call(this.input, value);
+    }
+
+    _createElements () {
+        this.element      = createElementFromDescriptor(descriptors.selectorInputContainer);
+        this.input        = createElementFromDescriptor(descriptors.selectorInput);
+        this.indicator    = createElementFromDescriptor(descriptors.matchIndicator);
+        this.expandButton = createElementFromDescriptor(descriptors.expandSelectorsList);
+
+        this.element.appendChild(this.input);
+        this.element.appendChild(this.indicator);
+        this.element.appendChild(this.expandButton);
+    }
+
+    _addEventListeners () {
+        eventUtils.bind(this.expandButton, 'click', () => this._expandSelectorsList());
```

Comment:
can we use the following approach?
eventUtils.bind(this.expandButton, 'click', this._expandSelectorsList.bind(this))
It's not required though